sonar-project.properties

```diff
@@ -4,7 +4,7 @@ sonar.sources=.
 sonar.tests=.
 sonar.test.inclusions=**/*.spec.ts
 sonar.exclusions=**/*.js,jest.config.ts,globalSetup.ts,globalTeardown.ts,**/*.app.ts,**/seed-data/*.ts
-sonar.coverage.exclusions=**/board-management.uc.ts,**/*.module.ts,**/*.factory.ts
+sonar.coverage.exclusions=**/modules/tldraw/gateway/tldraw.gateway.ts,**/modules/tldraw/utils/utils.ts,**/board-management.uc.ts,**/*.module.ts,**/*.factory.ts

```

Comment:
Please don't add not tested code to the nest server

Reply:
you can write a mock for the external dependencies these files are dealing with, to make it easier to test them. see the ldap.service.ts for an example.

And thanks for reminding me that the board-management is still excluded here, that has to be removed as well (it has been excluded when sonarcloud was first activated, since its not used in production anyway, but it still needs to go)